recipes-ni/initscripts-nilrt/files/populateconfig

```diff
@@ -0,0 +1,100 @@
+#!/bin/sh
+# Copyright (c) 2012-2013 National Instruments.
+# All rights reserved.
+
+if [ "$1" = "start" ]
+then
+	#check if the system is in reset state
+	reset_state=`/sbin/fw_printenv -n sys_reset 2>&1`
+	if [ "$reset_state" = "true" -o "$sys_reset" = "true" ]; then
+		#restore default environmnet vars
+		cp -fp /boot/grub/grubenv.bak /boot/grub/grubenv
+		chown root:ni /boot/grub/grubenv
+		chmod 664 /boot/grub/grubenv
+		/usr/local/natinst/bin/niresetip
+	fi
+
+	if [ -x /usr/local/natinst/bin/nirtcfg ]; then
```

Comment:
This file is run in run levels 0 and 6 too apparently, and only this part of the script makes sense to run at that time. So I left those runlevels as is.